libs/platform/auth/login-link/login-link.component.ts

```diff
@@ -31,13 +27,12 @@ export class LoginLinkComponent extends ContentMixin<LoginLinkOptions>(
     const i18nToken = this.$isAuthenticated() ? 'auth.logout' : 'auth.login';
 
     return html`
-      <oryx-button
-        .type=${ButtonType.Text}
-        .text=${this.i18n(i18nToken)}
-        .icon=${IconTypes.Login}
+      <oryx-site-dropdown-item
```

Comment:
Looks like a BC. Content element was changed that potentially can break custom styles. I would suggest to render an `oryx-button` for the < 1.1, and  `oryx-site-dropdown-item` for >= 1.1. Don't forgot to restore the styles and make them also conditional